runner/runners_test.go

```diff
@@ -227,21 +227,64 @@ func testRunners(t *testing.T, context spec.G, it spec.S) {
 
 	context("set default --target argument", func() {
 		it("is specified by the user", func() {
-			Expect(runner.AddDefaultTargetForTiny([]string{"install", "--target"}, "foo")).To(Equal([]string{"install", "--target"}))
-			Expect(runner.AddDefaultTargetForTiny([]string{"install", "--target=test"}, "foo")).To(Equal([]string{"install", "--target=test"}))
-			Expect(runner.AddDefaultTargetForTiny([]string{"install", "--target", "test"}, "foo")).To(Equal([]string{"install", "--target", "test"}))
+			Expect(runner.AddDefaultTargetForTinyOrStatic([]string{"install", "--target"}, "foo", runner.StaticTypeMUSLC)).To(Equal([]string{"install", "--target"}))
```

Comment:
I'm not sure what your testing philosophy is, but now that `AddDefaultTargetForTinyOrStatic` has two return arguments - the second being an error - it seems reasonable to explicitly assert the error is `nil` in all these test cases.

And, for bonus credit, it might be helpful to test the cases where error is not `nil` - i.e. when `os.Setenv` fails.

Reply:
Fixed.